### PR TITLE
counting word frequency from a job site

### PR DESCRIPTION
**Pull request by Kunal614**

### CODE REVIEW

File: count_freq_job.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            new_list.append(i)
+    return new_list
+
+
+url="https://www.roberthalf.com/"
+ua = {'UserAgent':'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:70.0) Gecko/20100101 Firefox/70.0'}
+res = requests.get(url,headers=ua)
+soup = BeautifulSoup(res.content,'html.parser')
+
+data = soup.find_all(class_='clearfix text-formatted field field--name-field-eck-carousel-text field--type-text-long field--label-hidden field__item')
```
Comment (stamarty) [inline count_freq_job.py:22]: Does this count words just on the main roberthalf page? Can I get a quick explanation of what's happening on these lines please? I think this looks really good though, and I thank you for the time you're putting into it.

Comment (Kunal614) [reply count_freq_job.py:22]: Thank you sir,
In line 17 , Url of the site is there , in 18 user agent of firefox is there , in line 19 it request the site to give his source code or simple language it ask to use that site source code , in line 20 it takes its html source code , and in line 22 it takes the valuable data.

Comment (Kunal614) [reply count_freq_job.py:22]: If you have more problems related to data scrapping please give me , i am very happy to work with you .

Comment (Kunal614) [reply count_freq_job.py:22]: Screenshot from 2019-11-18 17-25-53

It give dictionary as an output .

Comment (stamarty) [reply count_freq_job.py:22]: Thanks, buddy. I merged the PR. I appreciate the work you put into this. I won't get into testing it heavily until later this week, but I really appreciate it.

**Comment by Kunal614** [comment]
thanks , if you have more questions related to web scrapping then you free
to give me , I am always ready for doing this kind of stuffs.

**Comment by Kunal614** [comment]
you know about Gsoc ?

**Pull request merged**